.github/workflows/code-checks.yml

```
@@ -18,4 +18,7 @@ jobs:
         run: |
           grep mypy requirements/test.txt | xargs -0 pip install
           pip list
+          pip install -r requirements.txt
+          pip install “.[dev, examples]”
```

I think this should be enough, we don't run mypy on examples

```suggestion
          pip install '.[dev]'
```